src/corner-dialog/src/CornerDialog.js

```diff
@@ -143,7 +146,12 @@ export default class CornerDialog extends PureComponent {
     cancelLabel: 'Close',
     onCancel: close => close(),
     onConfirm: close => close(),
-    onCloseComplete: () => {}
+    onCloseComplete: () => {},
+    position: {
+      position: 'fixed',
+      bottom: 16,
+      right: 50
```

Comment:
curious why the change from `16` to `50` for the defaults here?

Reply:
Oh, I did that just for approach the box to the bottom. Sryy, I will back to 50.